src/kaboom.ts

```diff
@@ -4835,15 +4835,18 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
 			throw new Error("lifespan() requires time")
 		}
 		let timer = 0
+		let startOpacity = 0
```

Comment:
I'd make this a `null` and check for `=== null` later, to say that the value doesn't exist yet instead of is number 0